cmd/flow-aggregator/config.go

```diff
@@ -21,12 +21,21 @@ type FlowAggregatorConfig struct {
 	// If no L4 transport proto is given, we consider tcp as default.
 	// Defaults to "".
 	ExternalFlowCollectorAddr string `yaml:"externalFlowCollectorAddr,omitempty"`
-	// Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
-	// records to the flow collector.
-	// Flow export interval should be greater than or equal to 1s (one second).
-	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-	// Defaults to "60s".
-	FlowExportInterval string `yaml:"flowExportInterval,omitempty"`
+	// Provide the active flow record timeout as a duration string. This determines
+	// how often the flow aggregator exports the active flow records to the flow
+	// collector. Thus, for flows with a continuous stream of packets, a flow record
+	// will be exported to the collector once the elapsed time since the last export
+	// event is equal to the value of this timeout.
+	// Defaults to "60s". Valid time units are "ns", "us" (or "µs"), "ms", "s",
+	// "m", "h".
+	ActiveFlowRecordTimeout string `yaml:"activeFlowRecordTimeout,omitempty"`
+	// Provide the inactive flow record timeout as a duration string. This determines
+	// how often the flow aggregator exports the inactive flow records to the flow
+	// collector. A flow record is considered to be inactive if no matching record
+	// has been received from any Node in the cluster since the last export event.
```

Comment:
Here you mean since last export event from original exporter? Not export event of flow aggregator, right?

Reply:
Yes.

Reply:
I agree with @zyiou that this is a bit confusing. Maybe "A flow record is considered to be inactive if no matching record is received by the aggregator for the specified duration"

Reply:
Rephrased it.